extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.autoplay.js

```diff
@@ -17,10 +17,13 @@ define('wikia.articleVideo.featuredVideo.autoplay', [
 			return isAutoplayDisabledByRabbit;
 		},
 		isAutoplayEnabled: function () {
-			return featuredVideoCookieService.getAutoplay() !== '0' &&
+			var isEnabled =  featuredVideoCookieService.getAutoplay() !== '0' &&
 				// adContext.isEnabled('wgArticleVideoAutoplayCountries') &&
```

Comment:
Add `ToDo` why it is commented out?

Reply:
I didn't add a TODO as it was already here, probably a remnant of other team's experiment.